docs/reading_files.rst

```diff
@@ -156,6 +156,18 @@ data, so you use them like this::
 
    .. automethod:: union
 
+Selecting a single data source
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+One can select a single data source as a ``KeyData`` object. It offers the same
+functionality you can find in the ``DataCollection`` object. To select a data source
+from a run::
+
+  keydata = run[source, key]
+
+.. autoclass:: extra_data.keydata.KeyData
+   :members:
```

Comment:
I have a slight preference for listing the methods manually, because then we can put them in a meaningful order. E.g. at the moment, the first method is `dask_array`, which is probably not a good starting point for understanding the API. And it would be good to group `ndarray` and `xarray` together, as they're conceptually very similar.

Reply:
Good point, I'll update that